src/bin/index.js

```diff
@@ -329,25 +357,54 @@ yargs(hideBin(process.argv))
             ...commonOptions,
           },
           async (argv) => await lightnetExplorer(argv)
-        )
-        .demandCommand();
-    }
-  )
-  .version(
-    fs.readJsonSync(path.join(__dirname, '..', '..', 'package.json')).version
-  )
-  .alias('h', 'help')
-  .alias('v', 'version')
-
-  .epilog(
-    // chalk.reset is a hack to force the terminal to retain a line break below
-    chalk.reset(
-      `
-
-    █▄ ▄█ █ █▄ █ ▄▀▄
-    █ ▀ █ █ █ ▀█ █▀█
+        );
+    },
+  };
+}
 
-     Mina Protocol
-      `
-    )
-  ).argv;
+function getCustomizedStrings() {
+  // https://github.com/yargs/yargs/issues/199
+  // https://github.com/yargs/yargs/blob/master/locales/en.json
+  return {
+    'Not enough non-option arguments: got %s, need at least %s': {
+      one: chalk.red(
+        'Not enough non-option arguments: got %s, need at least %s'
```

Comment:
suggest same wording updates for these other arguments

Reply:
Thank you @barriebyron but I'll move all the text suggestions to another PR with only such improvements. There are other places that needs to be updated and it will be good to handle it all together.

Reply:
https://github.com/o1-labs/zkapp-cli/issues/592